src/app/components/plugins/plugins.component.html

```diff
@@ -26,5 +26,8 @@ <h3 class="plugin-filter-item">FILTER PLUGINS</h3>
     <mat-tab label="Events">Events Content</mat-tab>
     <mat-tab label="Messages">Message Content</mat-tab>
     <mat-tab label="Users">User Content</mat-tab>
+    <mat-tab label="Dashboard">
```

Comment:
Can we rename this to Grafana or Telematics

Reply:
Addressed https://github.com/usdot-fhwa-OPS/v2xhub-angular-ui-prototype/pull/1/commits/861d750f18b82e4210a4eba55055c0a7e7d71d90